substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/NativeImageDebugInfoProvider.java

```diff
@@ -42,6 +42,20 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import com.oracle.svm.core.graal.meta.RuntimeConfiguration;
+import com.oracle.svm.hosted.c.NativeLibraries;
+import com.oracle.svm.hosted.c.info.AccessorInfo;
+import com.oracle.svm.hosted.c.info.ElementInfo;
+import com.oracle.svm.hosted.c.info.PointerToInfo;
+import com.oracle.svm.hosted.c.info.PropertyInfo;
+import com.oracle.svm.hosted.c.info.RawStructureInfo;
+import com.oracle.svm.hosted.c.info.SizableInfo;
+import com.oracle.svm.hosted.c.info.SizableInfo.ElementKind;
+import com.oracle.svm.hosted.c.info.StructFieldInfo;
+import com.oracle.svm.hosted.c.info.StructInfo;
+import com.oracle.svm.util.ClassUtil;
+import jdk.vm.ci.code.CallingConvention;
+import jdk.vm.ci.meta.AllocatableValue;
```

Comment:
Nitpick: The ordering of this imports seems unaligned with the rest. How come checkstyle is not complaining?

Reply:
Hmm, I'm not sure why they are out of order nor why checkstyle is not complaining. I had to rebase the patch to deal with the addition of a heartbeat callback to the debug info generator and this seemed to be the order in head at the time of merge.

Reply:
I have re-ordered the imports so they are grouped alphabetically.